glean-core/android/src/main/java/mozilla/telemetry/glean/rust/LibGleanFFI.kt

```diff
@@ -81,6 +81,25 @@ internal interface LibGleanFFI : Library {
 
     fun glean_counter_test_has_value(glean_handle: Long, metric_id: Long, storage_name: String): Byte
 
+    fun glean_datetime_set(
+        glean_handle: Long,
+        metric_id: Long,
+        year: Int,
+        month: Int,
+        day: Int,
+        hour: Int,
+        minute: Int,
+        second: Int,
+        nano: Int, // FIXME?
```

Comment:
This should really be a `u32` :|

Reply:
I think are limited to the [types that JNI supports](https://docs.oracle.com/javase/7/docs/technotes/guides/jni/spec/types.html#wp428).  So maybe use `Long`/`i64` at the FFI layer (which is big enough to hold any `u32` value), and truncate to `u32` on the Rust side inside `glean_datetime_set`?

Reply:
Oh, and hopefully iOS has the same subset of numeric types available :)